src/app/modules/submission/email-submission/email-submission.util.ts

```diff
@@ -425,6 +430,24 @@ export const mapRouteError: MapRouteError = (error) => {
         statusCode: StatusCodes.BAD_REQUEST,
         errorMessage: 'Captcha was missing. Please refresh and submit again.',
       }
+    case TurnstileConnectionError:
+      return {
+        statusCode: StatusCodes.INTERNAL_SERVER_ERROR,
+        errorMessage:
+          'Error connecting to Turnstile server . Please submit again in a few minutes.',
+      }
+    case VerifyTurnstileError:
+      return {
+        statusCode: StatusCodes.INTERNAL_SERVER_ERROR,
```

Comment:
It might make sense to have this as client error of `StatusCodes.BAD_REQUEST` instead of server error? Since the server was able to interpret the response but the response from client was invalid.

Reply:
Hey, thanks for the catch. I was deciding between 500 and 400 but now I think 400 makes more sense given than malicious users might try to fake the token and cause an invalid request. Amended.